.up/kcl/models/com/example/platform/v1alpha1/storagebucket.k

```diff
@@ -37,37 +37,49 @@ schema StorageBucket:
 
 schema PlatformExampleComV1alpha1StorageBucketSpec:
     r"""
-    platform example com v1alpha1 storage bucket spec
+    StorageBucketSpec defines the desired state of StorageBucket.
 
     Attributes
     ----------
+    compositeDeletePolicy : str, default is Undefined, optional
+        composite delete policy
     compositionRef : PlatformExampleComV1alpha1StorageBucketSpecCompositionRef, default is Undefined, optional
         composition ref
     compositionRevisionRef : PlatformExampleComV1alpha1StorageBucketSpecCompositionRevisionRef, default is Undefined, optional
         composition revision ref
+    compositionRevisionSelector : PlatformExampleComV1alpha1StorageBucketSpecCompositionRevisionSelector, default is Undefined, optional
+        composition revision selector
     compositionSelector : PlatformExampleComV1alpha1StorageBucketSpecCompositionSelector, default is Undefined, optional
         composition selector
-    compositionUpdatePolicy : str, default is "Automatic", optional
+    compositionUpdatePolicy : str, default is Undefined, optional
```

Comment:
Is this correct?

Reply:
Please check: https://github.com/upbound/up/pull/286/files#r1847172010